client/lib/scroll-to/test/index.js

```diff
@@ -15,7 +15,7 @@ describe( 'scroll-to', () => {
 	useFakeDom();
 
 	before( () => {
-		scrollTo = require( '..' );
+		scrollTo = require( '..' ).default;
```

Comment:
@stephanethomas This change breaks the tests. It appears to work properly without this change, though.